docs/configuration.md

```diff
@@ -753,7 +753,7 @@ interpretation of the options.
       no
     </td>
     <td>
-      Duration for which a signed URL should be valid.
+      Specify a `duration` by provding an integer and a unit. Valid time units are `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`. For example, `3m` is a valid duration. If you omit this value or do not specify a time unit, this defaults to 20 minutes.
```

Comment:
The only case that works is `integer + unit` anything else causes the system to use the default.  I've made this more explicit.